BraveRewardsUI/Settings/Auto-Contribute/Details/AutoContributeDetailsViewController.swift

```diff
@@ -63,25 +44,44 @@ class AutoContributeDetailViewController: UIViewController {
     
     navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .edit, target: self, action: #selector(tappedEditButton))
     
-    reloadData()
   }
   
   override func viewWillAppear(_ animated: Bool) {
     super.viewWillAppear(animated)
-    
     reloadData()
   }
   
   func reloadData() {
-    // FIXME: Remove temp values
     let dateFormatter = DateFormatter().then {
       $0.dateStyle = .short
       $0.timeStyle = .none
     }
-    nextContributionDateView.label.text = dateFormatter.string(from: Date().addingTimeInterval(60*60*24*12))
+    let reconcileDate = Date(timeIntervalSince1970: TimeInterval(ledger.autoContributeProps.reconcileStamp))
+    nextContributionDateView.label.text = dateFormatter.string(from: reconcileDate)
     nextContributionDateView.bounds = CGRect(origin: .zero, size: nextContributionDateView.systemLayoutSizeFitting(UIView.layoutFittingExpandedSize))
+    loadPublishers(start: 0) {[weak self] publishersList in
+      self?.publishers = publishersList
+      self?.hasMoreContent = publishersList.count == AutoContributeDetailViewController.pageSize
+      self?.contentView.tableView.reloadData()
+    }
+  }
+  
+  private func loadPublishers(start: Int, limit: Int  = AutoContributeDetailViewController.pageSize, completion: @escaping ([PublisherInfo]) -> Void) {
+    let sort = ActivityInfoFilter.OrderPair().then {
+      $0.propertyName = "percent"
+      $0.ascending = false
+    }
+    let filter = ActivityInfoFilter().then {
+      $0.id = ""
+      $0.excluded = .filterAllExceptExcluded
+      $0.percent = 1 //exclude 0% sites.
```

Comment:
Is 1 the lowest? Or should we simply be filtering anything that actually equals 0? Not sure if 0.5 is valid or not

Reply:
I verified with brave-core, the lowest they show is 1.